### PR TITLE
[TACHYON-1460] Add integration test for the OSS UnderFileSystem

### PR DESCRIPTION
**Pull request by luoli523**
Add the OSSUnderStorageCluster for OSSUnderFilesystem integration test.

we should config the pom.xml in integration-test, change four properties below:
1, fs.oss.accessKeyId (the oss access id)
2, fs.oss.accessKeySecret (the oss access secret key)
3, fs.oss.endpoint (because the uri endpoint are different from range to another range,eg: hangzhou, shanghai, Singapore, HongKong...)
4, ossBucket (the bucket address when the tests run into)

The integration test of OSS is much like the test of S3UnderFileSystem,  but I noticed that in the current master ， the integration tests for S3 cannot run successfully. I use my own aws id and key to run and keeps fail like below:

```java
java.lang.IllegalArgumentException: All eligible Under File Systems were unable to create an instance for the given path: s3n://my-bucket/tachyon-testc23eb0b5-4b3a-42e5-a481-1f2454574b93/journal-290143156907472538
java.lang.RuntimeException: Invalid configuration key fs.s3n.awsAccessKeyId

	at tachyon.underfs.UnderFileSystemRegistry.create(UnderFileSystemRegistry.java:132)
	at tachyon.underfs.UnderFileSystem.get(UnderFileSystem.java:106)
	at tachyon.underfs.UnderFileSystem.get(UnderFileSystem.java:90)
	at tachyon.util.UnderFileSystemUtils.mkdirIfNotExists(UnderFileSystemUtils.java:54)
	at tachyon.master.AbstractLocalTachyonCluster.setupTest(AbstractLocalTachyonCluster.java:251)
	at tachyon.master.AbstractLocalTachyonCluster.start(AbstractLocalTachyonCluster.java:105)
	at tachyon.LocalTachyonClusterResource.apply(LocalTachyonClusterResource.java:167)
```

I think this is probably because the reconstruction of the integration test and the LocalTachyonClusterResource TestRule using. I encounter the same problem when I test OSS. So the integration test of OSS is a little different to the test of S3.

To run the test, people should have a aliyun oss account and get the accessid and accessKey, I have a temp id and key pair, people who wana to use please just require it from me.

After the config done, run the test like below:

    cd integration-tests
    mvn clean -Dtest=UnderStorageSystemInterfaceIntegrationTest -DfailIfNoTests=false test -PossTest

Please confirm this.

### CODE REVIEW

**Comment by apc999** [comment]
+1 LGTM

**Comment by luoli523** [comment]
@yupeng9 @apc999 can this patch merged to the oss branch?

File: integration-tests/pom.xml [hunk 1]
```diff
@@ -206,6 +206,45 @@
...
+              <systemPropertyVariables>
+                <ufs>tachyon.underfs.oss.OSSUnderStorageCluster</ufs>
+                <!-- Change "myAccessId" to your oss access id for this test -->
+                <fs.oss.accessKeyId>myAccessId</fs.oss.accessKeyId>
+                <!-- Change "mySecretKey" to your oss access secret key for this test -->
+                <fs.oss.accessKeySecret>mySecretKey</fs.oss.accessKeySecret>
+                <!-- Change this endpoint for your oss service address for this test -->
+                <fs.oss.endpoint>http://oss-cn-hangzhou.aliyuncs.com</fs.oss.endpoint>
+                <!-- Change "your-bucket" to your oss bucket for this test -->
+                <ossBucket>oss://your-bucket/tachyon-integration-test/</ossBucket>
```
Comment (yupeng9) [inline integration-tests/pom.xml]: NIT: For consistency, could you either change this to `my-bucket` or rename to `yourAccessId` and `yourSecretKey`?

**Comment by yupeng9** [comment]
Thanks for adding the integration tests. I left one minor comment.

Also, thanks for reporting the issue with s3. I'll take a look.

**Comment by luoli523** [comment]
@yupeng9   I updated the PR  to address your suggestion. 
thanks a lot to review.
Please check it again.

also, the issue about s3, I recorded here,please confirm it.
https://tachyon.atlassian.net/browse/TACHYON-1469

**Comment by apc999** [comment]
Jenkins, test this please.

**Comment by luoli523** [comment]
@apc999 OMG, Jenkins is real?

**Comment by yupeng9** [comment]
LGTM. Thanks!

**Comment by apc999** [comment]
@luoli523 , There is a  Jenkins instance running for this repo :)

**Pull request merged**